src/__tests__/__snapshots__/Interpreter.test.js.snap

```diff
@@ -530,6 +544,20 @@ Object {
       "type": "ERC20",
     },
   ],
+  "chainSymbol": "ETH",
+  "contractAddress": "0x68b3465833fb72a70ecdf485e0e4c7bd8665fc45",
+  "contractName": "Uniswap V3",
+  "counterpartyName": null,
+  "exampleDescription": "0x5596 swapped 0.000000280376616698 USDC for 150.286887655178993883 WETH on Uniswap V3",
```

Comment:
Example of USDC being 12 decimals off b/c it uses 6 instead of 18... no clean way really to fix this here since we don't know it's USDC until the time of interpretation and we only look at event logs, not the assetsSent / assetsReceived arrays